README.rst

```diff
@@ -94,7 +94,7 @@ To see what coala can do for your language, run:
 +----------------------------+----------------------------+----------------------------+
 | Fortran                    | Python 2                   | YAML                       |
 +----------------------------+----------------------------+----------------------------+
-| Go                         | Python 3                   |                            |
+| Go                         | Python 3                   | Stylus                     |
```

Comment:
could you try to maintain alphabetical order? I know we have chosen a quite unsuited layout though 😅